devdoc/constbuffer_requirements.md

```diff
@@ -98,6 +104,8 @@ MOCKABLE_FUNCTION(, CONSTBUFFER_HANDLE, CONSTBUFFER_Create, const unsigned char*
 
 **SRS_CONSTBUFFER_02_005: [** The non-NULL handle returned by `CONSTBUFFER_Create` shall have its ref count set to "1". **]** 
 
+**SRS_CONSTBUFFER_51_015: [** Shall set the state of `CONSTBUFFER_HANDLE` to `CONSTBUFFER_WRITIABLE_STATE_SEAL`. **]**
```

Comment:
>Shall [](http://example.com/codeflow?start=30&length=5)

We put the function name in the spec:
`CONSTBUFFER_Create` shall set the state ...

Reply:
Here and everywhere

Reply:
Added prefix all the places .